scalafix/rules/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -101,31 +141,40 @@ case class Collectionstrawman_v0(index: SemanticdbIndex)
       i("`+:`"),
       s("`:+`"),
       i("`:+`"),
-      i("Stream", Some("LazyList")),
       s("Stream", Some("LazyList")),
-      s("Traversable", Some("Iterable")),
```

Comment:
I believe the one above was incorrect. It rewrites `scala.collection.Traversable` to `strawman.collection.immutable.Iterable`. I assume the correct import would be `strawman.collection.Iterable`

Reply:
Indeed, I thought `scala.Traversable[A] = scala.collection.immutable.Traversable[A]` but I was wrong.